idaes_examples/notebooks/docs/surrogates/sco2/omlt/surrogate_embedding_usr.ipynb

```diff
@@ -443,16 +444,30 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "Now we have our property package ready for being used in the flowsheet for optimization. We shall see that in the next part of this tutorial, [flowsheet_optimization](./flowsheet_optimization_usr.ipynb). To learn in detail about making a custom property package, one should go through [Property Package Example](../../../properties/custom/custom_physical_property_packages_usr.ipynb). "
```

Comment:
@bpaul4 was this change required because of a specific error? Usually I'd expect (but haven't actually checked) that Unix-style paths with `/` are accepted on Windows as well, so I'd be curious to know if there might be something that warranted this change.

Reply:
That's very odd. The "\\" change doesn't exist on my local branch; there's another change in "_doc" version of this notebook from ".ipynb" to ".md" that I am sure I did not make. Not sure how this happened, as the other versions should all be identical to the source notebook, correct?

Reply:
I agree, that's very strange. I'll try to look into it, starting from attempting to reproduce this locally.